apps/dc_tools/odc/apps/dc_tools/index_from_tar.py

```diff
@@ -51,6 +181,8 @@ def untar(tarfname, mk_uri):
 @click.option('--xz', is_flag=True, help='Input is compressed with xz (needed when reading from stdin)')
 @click.option('--protocol', type=str, default='s3', show_default=True,
               help='Override the protocol for working with data in other environments, i.e gs')
+@click.option('--stac', is_flag=True, default=False, show_default=True,
+              help='Convert STAC 1.0 metadata to ODC EO3 metadata')
```

Comment:
Maybe re-word this a bit? "Expect STAC 1.0 metadata on input" or something along these lines.